interfaces.go

```diff
@@ -65,24 +65,35 @@ type Statuser interface {
 // (approximate) minimum of the objective function along the search direction
 // dir_k starting at the most recent location x_k, i.e., it tries to minimize
 // the function
+//
```

Comment:
I like the formulas to have some margin around them, similarly to mathematical formulas in papers. It looks better to me and the formula stands out, which is often good, for example in this case we refer to it at several places below. But ok, I removed the newlines.